auth/store.go

```diff
@@ -1070,7 +1070,6 @@ func NewAuthStore(lg *zap.Logger, be backend.Backend, tp TokenProvider, bcryptCo
 
 	if as.Revision() == 0 {
 		as.commitRevision(tx)
-		as.saveConsistentIndex(tx)
```

Comment:
Just want to make sure the motivation of dropping this line. `NewAuthStore()` isn't called as a command apply so you are dropping it, right?

Reply:
yes, it will also cause the error log to be printed because syncConsitentIndexFunc is nil at this time.

Reply:
Got it, then it's fine :+1: